var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -220,6 +220,11 @@ def url_for_version(self, version):
         url = "http://www.open-mpi.org/software/ompi/v{0}/downloads/openmpi-{1}.tar.bz2"
         return url.format(version.up_to(2), version)
 
+    @property
+    def headers(self):
+        hdrs = HeaderList(find(self.prefix.include, 'mpi.h', recursive=False))
```

Comment:
For external packages, we need to search `lib` as well (see #7268).

Reply:
OK, I'll add a fall-back that searches all of `prefix` recursively.